backend/apiServer/model/VariableModel.go

```diff
@@ -0,0 +1,7 @@
+package model
+
```

Comment:
Do you handle global variables? can only see them in functionModel not FileModel.

Reply:
no, haven't done it but i can make it happen.